src/features/landing/components/ForWhoSection/ForWhoSection.tsx

```
@@ -1,26 +1,26 @@
 function ForWhoSection(): React.ReactElement {
-  const iconAltText: string = 'Vector';
-
-  const tabletProps: ImageProps = getOptimizedImageProps({
-    src: TabletVectorIcon,
-    alt: iconAltText,
-  }).props;
-  const desktopProps: ImageProps = getOptimizedImageProps({
-    src: DesktopVectorIcon,
-    alt: iconAltText,
-  }).props;
+  const getImageProps: (src: string, alt?: string) => React.ImgHTMLAttributes<HTMLImageElement> = (src, alt = "") => 
```

ESLint

[error] Strings must use singlequote.

(quotes)